### PR TITLE
set executemany to false for update statements

### LINKED ISSUE #36: Exasol Dialect should not execute update statements with executemanny
In EXAExecutionContext the setting executemany  is set to True. Therefore update statements are also executed via executemanny(..). 
While executing the update statenment works, the result.rowcount field is always set to -1 which breaks alembic > 0.7 https://bitbucket.org/zzzeek/alembic/src/a192c345fcfcf4d5dd817f898ec883d7fa3b0ac4/alembic/migration.py?at=master#cl-459 

```python
from sqlalchemy import *
dsn = "exa+pyodbc://scott:tiger@host:8563/"
engine = create_engine(dsn, echo=True)

md = MetaData(bind=engine)
md.reflect(schema="test")
t = md.tables.get("test.alembic_version")
update_statement = update(t).values(version_num = 'dc0a2e2ac10')
r = engine.execute(update_statement)
print "Rowcount %s" %r.rowcount
```

```
2015-04-29 20:09:04,992 INFO sqlalchemy.engine.base.Engine (u'TEST',)
2015-04-29 20:09:05,208 INFO sqlalchemy.engine.base.Engine UPDATE test.alembic_version SET version_num=?
2015-04-29 20:09:05,208 INFO sqlalchemy.engine.base.Engine (('dc0a2e2ac10',),)
2015-04-29 20:09:05,300 INFO sqlalchemy.engine.base.Engine COMMIT
Rowcount -1
```

### PR DESCRIPTION
**Pull request by hoffmann**
Fixes #36

### CODE REVIEW

**Comment by kopf** [comment]
Great that you managed to track this bug down! 

I'm not 100% familiar with `executemany`. Will turning it off for all `UPDATE`s not incur a performance hit?

**Comment by hoffmann** [comment]
I'm not aware of cases where we really use bindparm syntax to do multiple updates at once http://docs.sqlalchemy.org/en/latest/core/tutorial.html#inserts-updates-and-deletes

**Comment by kopf** [comment]
:+1:

File: test/test_update.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    def fixtures(cls):
+        return dict(
+            users=(
+                ('id', 'name'),
+                (7, 'jack'),
+                (8, 'ed'),
+                (9, 'fred'),
+                (10, 'chuck')
+            ),
+            addresses = (
```
Comment (kopf) [inline test/test_update.py]: No spaces between kwargs and `=` (or, better yet, use `{}` instead of `dict()`)

**Comment by kopf** [comment]
OK, looks fine. All I can suggest is adding some docstrings to the tests, as it's not immediately completely clear what they're testing.

**Pull request merged**